x-pack/plugins/aiops/public/components/log_categorization/log_categorization_for_embeddable/log_categorization_for_embeddable.tsx

```diff
@@ -354,12 +354,17 @@ export const LogCategorizationEmbeddable: FC<LogCategorizationEmbeddableProps> =
     [lastReloadRequestTime]
   );
 
-  const actions = [...getActions(false), ...getActions(true)];
+  const isCasesEmbeddable = embeddingOrigin === AIOPS_EMBEDDABLE_ORIGIN.CASES;
+
+  // Disable filtering for cases embeddable, as it's not possible to delete filters
+  const actions = [...(isCasesEmbeddable ? [] : getActions(false)), ...getActions(true)];
```

Comment:
It disables filtering for a pattern in cases. In dashboards, it is possible to delete a filter, but there is currently no such option in cases. I have disabled it for now, but it would be nice to have that option available inside the embeddable.